CS Falcon: get detections from right key (#36341)
demisto/content: Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_18.md

#### Integrations

##### CrowdStrike Falcon

> - Fixed an issue where **detection_ids** are missed in the ***cs-falcon-get-detections-for-incident*** command results.

Reviewer: ```suggestion
- Fixed an issue where **detection_ids** were missed in legacy version ***cs-falcon-get-detections-for-incident*** command results.
```
